types/dialogify-browser/global.d.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="index.d.ts" />
+
+export {};
+
+declare global {
```

Comment:
please add this to `index.d.ts`. Original module has no `global` file, thx!

Reply:
I would be willing to do so, but if I add this to `index.d.ts`, it will have following error:

```
Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.ts(2669)
```

Most solutions come from internet told me I have to change it to a module... Will c3d4626 better? Or what should I do? Is there any type definition I can study?